src/qtbase-1-fixes.patch

```diff
@@ -221,6 +221,7 @@ index 4dd9e9603b..c94d64d9aa 100644
 -- 
 2.30.2
 
+
```

Comment:
stray whitespace

Reply:
No it's not. All patches area separated by double newlines, and this one was missing that, so I added it.  But I can remove that fix if it matters.